### PR TITLE
Add influxdb cluster variant documentation

### PR DESCRIPTION
**Pull request by jsternberg**
Documentation for influxdata/influxdata-docker#73.

### CODE REVIEW

File: influxdb-meta/README-short.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+InfluxDB Enterprise is a highly available time series database.
```
Comment (pauldix) [inline influxdb-meta/README-short.txt]: InfluxDB Enterprise is the commercial version of the open source InfluxDB, offering high availability and horizontal scalability.

**Review by pauldix — CHANGES_REQUESTED** [review]

File: influxdb-meta/README-short.txt (hunk 1 above)
Comment (jsternberg) [reply influxdb-meta/README-short.txt]: They want the little blurb to be <= 100 characters. Is there a way we can remove some of the words and get the same point across?

Comment (pauldix) [reply influxdb-meta/README-short.txt]: InfluxDB Enterprise is the commercial offering of InfluxDB, with clustering and high availability.

99 characters, BOOM!

Comment (jsternberg) [reply influxdb-meta/README-short.txt]: Done.

**Pull request merged**